File: 3019-furthest-point-from-origin/solution.c
int furthestDistanceFromOrigin(char* moves) {
    int _count=0, leftCount=0, rightCount=0;
    for( int i=0; moves[i]!='\0'; i++ )
    {
        if( moves[i] == 'L' )
        leftCount++;
        else if( moves[i] == 'R' )
        rightCount++;
        else
        _count++;
    }
    if( rightCount >= leftCount )
        return rightCount - leftCount + _count;
    else
        return leftCount - rightCount + _count;
}

File: 3019-furthest-point-from-origin/solution.py
class Solution:
    def furthestDistanceFromOrigin(self, moves: str) -> int:
        _count = 0
        right_count = 0
        left_count = 0
        for i in moves:
            if i == 'R':
                right_count += 1
            elif i == 'L':
                left_count += 1
            else:
                _count += 1
        if right_count > left_count:
            return right_count + _count - left_count
        return left_count + _count -right_count
